test/avi/test_helpers.clj

```diff
@@ -117,23 +121,25 @@
     (first coll)
     coll))
 
-(defn line
-  [line expected]
+(defn line-cmp
+  [line expected ncmpBuf]
```

Comment:
Prefer all lower case for identifiers (except for protocol names or Java interop).